### PR TITLE
Feature/scale

### PR DESCRIPTION
**Pull request by rodrigopavezi**
- Added settings
- Implemented scale css
- Created tests

@donnapep @stulees please review.

Presentation here:
http://rva.risevision.com/#/PRESENTATION_MANAGE/id=1c8c52b8-ef13-4654-a58a-3102441f8fde?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

### CODE REVIEW

**Comment by stulees** [comment]
@rodrigopavezi :+1:

File: src/widget/webpage.js [hunk 1]
```diff
@@ -77,6 +78,30 @@ RiseVision.WebPage = (function (document, gadgets) {
...
+     zoomStyle;
+
+    // Configure the zoom (scale) styling
+    zoomStyle = "-ms-zoom:" + zoom + ";" +
+      "-moz-transform: scale(" + zoom + ");" +
+      "-moz-transform-origin: 0 0;" +
+      "-o-transform: scale(" + zoom + ");" +
+      "-o-transform-origin: 0 0;" +
+      "-webkit-transform: scale(" + zoom + ");" +
+      "-webkit-transform-origin: 0 0;";
```
Comment (donnapep) [inline src/widget/webpage.js]: @rodrigopavezi We should also add the unprefixed `transform-origin` as the last CSS property here.

Comment (rodrigopavezi) [inline src/widget/webpage.js]: @donnapep good point. I've added it.

**Comment by donnapep** [comment]
👍

**Pull request merged**